understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
     <section id="brief">
 			<h2>In brief</h2>
 			<dl>
-				<dt>Goal</dt><dd>Content can be enlarged without increasing line length.</dd>
-				<dt>What to do</dt><dd>Make lines of text reflow within the viewport.</dd>
-        <dt>Why it's important</dt><dd>People who need bigger text find it difficult if they must scroll to read long lines.</dd>
+				<dt>Goal</dt><dd>Content can be enlarged without needing to scroll in two dimensions.</dd>
+				<dt>What to do</dt><dd>Make long lines of text and other applicable content reflow within resized or zoomed in interfaces.</dd>
+        <dt>Why it's important</dt><dd>People who need larger text find it difficult if they need to scroll in two dimensions to read long lines that do not reflow.</dd>
 			</dl>
-
 		</section>
+  
     <section id="intent">
```

Comment:
adding this comment for myself to make sure i at least have a clear version of Mike's text from his [closed PR](https://github.com/w3c/wcag/pull/4126):

>  The most crucial problem solved by reflow is allowing users to increase text size while keeping line length
    within the width of the display. If users are forced to scroll to read lines of text, they are more likely to
    lose their place, especially when scrolling the opposite direction to locate the start of the next line.